src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -220,11 +219,13 @@ var Stream = React.createClass({
           loading: false
         });
       },
-      complete: () => {
+      complete: (jqXHR) => {
         this.lastRequest = null;
 
+        var links = parseLinkHeader(jqXHR.getResponseHeader('Link'));
+        this._poller.setEndpoint(links.previous.href);
+
         if (this.state.realtimeActive) {
-          this._poller.setEndpoint(url);
```

Comment:
Yes. Among other things, if realtime wasn't activated at the time when the first request finished, it would never get the new URL.